src/qt/askpassphrasedialog.cpp

```diff
@@ -234,6 +235,15 @@ bool AskPassphraseDialog::event(QEvent *event)
     return QWidget::event(event);
 }
 
+void AskPassphraseDialog::setShowPass(bool showPass)
+{
+    ui->passShow->setDown(showPass);
```

Comment:
Why?

Reply:
You mean why setDown()?  That is to show that it is in the 'active' state when the passwords are visible.  When you toggle back to masking the passwords, the button returns to the original up state.